src/test/mediapipeflow_test.cpp

```diff
@@ -2100,3 +2107,271 @@ INSTANTIATE_TEST_SUITE_P(
+    ConstructorEnabledModelManager manager;
+    std::string testPbtxt = R"(
+    input_stream: "in"
+    output_stream: "out"
+        node {
+            name: "pythonNode2"
+            calculator: "PythonBackendCalculator"
+            input_side_packet: "PYOBJECT:pyobject"
+            input_stream: "in"
+            output_stream: "out2"
+            node_options: {
+                [type.googleapis.com / mediapipe.PythonBackendCalculatorOptions]: {
+                    handler_path: "/ovms/src/test/mediapipe/python/script.py"
+                }
+            }
+        }
+    )";
+
+    ovms::MediapipeGraphConfig mgc{"mediaDummy", "", ""};
+    DummyMediapipeGraphDefinition mediapipeDummy("mediaDummy", mgc, testPbtxt);
+    mediapipeDummy.inputConfig = testPbtxt;
+    ASSERT_EQ(mediapipeDummy.validate(manager), StatusCode::OK);
+    std::unique_ptr<NodeState> nodeState = std::make_unique<NodeState>();
+    ASSERT_EQ(mediapipeDummy.getPythonNodeState("pythonNode4", nodeState), StatusCode::PYTHON_NODE_NAME_DOES_NOT_EXISTS);
+}
+
+TEST_F(MediapipePythonNodeTest, PythonNodeInitMembers) {
+    // Must be here - does not work when added to test::SetUp
+    // Initialize Python interpreter
+    py::scoped_interpreter guard{};  // start the interpreter and keep it alive
+    py::gil_scoped_release release;  // GIL only needed in Python custom node
+    ConstructorEnabledModelManager manager;
+    std::string testPbtxt = R"(
+    input_stream: "in"
+    output_stream: "out"
+        node {
+            name: "pythonNode2"
+            calculator: "PythonBackendCalculator"
+            input_side_packet: "PYOBJECT:pyobject"
+            input_stream: "in"
+            output_stream: "out2"
+            node_options: {
+                [type.googleapis.com / mediapipe.PythonBackendCalculatorOptions]: {
+                    handler_path: "/ovms/src/test/mediapipe/python/script.py"
+                }
+            }
+        }
+    )";
+
+    ovms::MediapipeGraphConfig mgc{"mediaDummy", "", ""};
+    DummyMediapipeGraphDefinition mediapipeDummy("mediaDummy", mgc, testPbtxt);
+    mediapipeDummy.inputConfig = testPbtxt;
+    ASSERT_EQ(mediapipeDummy.validate(manager), StatusCode::OK);
+    std::unique_ptr<NodeState> nodeState = std::make_unique<NodeState>();
+    ASSERT_EQ(mediapipeDummy.getPythonNodeState("pythonNode2", nodeState), StatusCode::OK);
+
+    try {
+        py::gil_scoped_acquire acquire;
+        using namespace py::literals;
+        py::module_ sys = py::module_::import("sys");
+        py::object model_instance = py::cast<py::object>(nodeState.get()->pythonNodeState);
+
+        // Casting and recasting needed for ASSER_EQ to work
+        py::str model_name = model_instance.attr("model_name");
+        py::str expected_name = py::str("testModel");
+        std::string s_model_name = model_name.cast<std::string>();
+        std::string s_expected_name = expected_name.cast<std::string>();
```

Comment:
Maybe we can reduce that to:
 std::string model_name = model_instance.attr("model_name").cast<std::string>();
 std::string expected_name = py::str("testModel").cast<std::string>();